packages/dd-trace/src/config.js

```diff
@@ -122,4 +122,14 @@ class Config {
   }
 }
 
+function isTrue (str) {
+  str = String(str).toLowerCase()
+  return str === 'true' || str === '1'
```

Comment:
worth adding `on`/`off` support?

I know some languages use that as well.

Reply:
That can be added in another PR. For now I focused on what we generally support.

Reply:
You could also use https://www.npmjs.com/package/yn, I think it's a tiny package